### DISCUSSION TITLE
[Feature Request] Add categories to profile repositories

### DISCUSSION THREAD

**Discussion by lulasz** [community/community#39360] [Repositories]
### Select Topic Area

Product Feedback

### Body

This is a feature request. I think it would be good, to have the ability to group user repositories using custom user made categories.

**Comment by Preetiraj3697** [comment]
@lulasz This is great idea. I also want this. 😊

**Comment by DSDmark** [comment]
Hello Mr.@lulasz,

Add ```[features request]``` in your **title** and also add **feed** in your category. 

It catches attention of GitHub team here.
